dockerfiles/verify_packages/verify_rpm_centos6.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "${PHP_INSTALL_DIR}" ]; then
+    echo "Please set PHP_INSTALL_DIR"
+    exit 1
+fi
+
+for phpVer in $(ls ${PHP_INSTALL_DIR}); do
+    echo "Installing ddtrace on PHP ${phpVer}..."
+    switch-php $phpVer
+    rpm -Uvh /build_src/build/packages/*.rpm
+    php --ri=ddtrace
+
+    # Uninstall the tracer
+    rpm -e datadog-php-tracer
+    rm -f /opt/datadog-php/etc/ddtrace.ini
```

Comment:
Question: should we remove the `9(8|9)-ddtrace.ini`s symlinks as well?

Reply:
Since we're just validating that it installs properly on each of the PHP flavors, it's not necessary to remove the symlinks. We're only removing the `/opt/datadog-php/etc/ddtrace.ini` file here because the post install hook does not overwrite an existing one (which would be pointing to the wrong `.so` after the second install).